pyquil/tests/test_paulis.py

```diff
@@ -484,7 +486,7 @@ def test_check_commutation_rigorous():
 
             tmp_op = _commutator(pauli_ops_pq[x], pauli_ops_pq[y])
             assert len(tmp_op.terms) == 1
-            if is_identity(tmp_op.terms[0]):
+            if is_zero(tmp_op.terms[0]):
```

Comment:
I believe so. If two operators commute, then their commutator equals the zero operator, not the identity. This test was previously passing because `is_identity(..)` used to (before these changes) falsely identify zero operators as identity operators as well, e.g. previously, `is_identity(PauliTerm("I", 0, 0.0))` would return `True`, but now (correctly) returns `False`.

Reply:
👍